corehq/apps/fixtures/download.py

```diff
@@ -357,19 +359,23 @@ def _load_location_codes(self, owners):
         return dict(codes)
 
     def count(self, row, owner_type):
-        return len(self.owners[row.id][owner_type])
+        names = self.names[owner_type]
+        return sum(1 for doc_id in self.owners[row.id][owner_type] if doc_id in names)
```

Comment:
Why would a doc exist in `self.owners[row.id][owner_type]` but not in `self.names[owner_type]`?

Reply:
> owners that have been deleted, but are still present in LookupTableRowOwner

might be the answer I'm looking for.